### PR TITLE
fix(package): build docs on start

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,10 +22,11 @@
...
   },
   "scripts": {
-    "start": "npm run update-data-sources && node --enable-source-maps build/app.js",
-    "start:server": "pm2 start ecosystem.config.cjs --env production",
-    "build": "tsc",
     "build:watch": "tsc -w",
+    "build": "tsc",
+    "regenerate-docs": "node --trace-warnings ./build/scripts/regenerate-docs.js",
+    "start:server": "pm2 start ecosystem.config.cjs --env production",
+    "start": "npm run build && npm run update-data-sources && npm run regenerate-docs && node --enable-source-maps build/app.js",
```
Comment (sidvishnoi) [inline package.json:29]:
```suggestion
    "prestart": "npm run build && npm run update-data-sources && npm run regenerate-docs",
    "start": "node --enable-source-maps build/app.js",
```

File: scripts/regenerate-docs.ts [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import { regenerateDocs } from "../routes/docs/update.js";
 
-(async () => {
-  const start = Date.now();
-  console.log("Regenerating docs...");
+const start = Date.now();
+console.log("Regenerating docs...");
+
+try {
   await regenerateDocs();
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply scripts/regenerate-docs.ts:7]: Not sure... but why are we using 14 on the serve? Server is ours, we should use 16++!!! :D

File: views/w3c/groups.ts [hunk 3]
```diff
@@ -62,13 +62,13 @@ export default ({ groups }: { groups: GroupsByType }) => html`
         List of possible values for
         <a href="/docs/#group"><code>respecConfig.group</code></a>.
       </p>
-      <div class="tables">
+      <section class="tables">
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply views/w3c/groups.ts:65]: Yeah, will split them out.

[the remark this answers is not in the document]
Comment (marcoscaceres) [reply views/w3c/groups.ts:65]: Gotta stop doing/sending multiple things at once ... angry at myself.

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**